### PR TITLE
fix #67516: add line with keyboardPedalUp to palette

### PR DESCRIPTION
**Pull request by MarcSabatella**
This comes up fairly often, and it's next to impossible to figure out how to do this yourself.

### CODE REVIEW

File: mscore/menus.cpp [hunk 1]
```diff
@@ -872,6 +872,16 @@ Palette* MuseScore::newLinesPalette(bool basic)
             pedal->setBeginText("<sym>keyboardPedalPed</sym>");
             pedal->setEndHook(true);
             sp->append(pedal, QT_TRANSLATE_NOOP("Palette", "Pedal"));
+
+            pedal = new Pedal(gscore);
+            pedal->setLen(w);
+            pedal->setBeginText("<sym>keyboardPedalPed</sym>");
+            pedal->setEndText("<sym>keyboardPedalUp</sym>");
```
Comment (lasconic) [inline mscore/menus.cpp]: According to BB p333, the continue text should be (Ped.)

Comment (MarcSabatella) [inline mscore/menus.cpp]: Ah, thanks, I did look but didn't see that colum in the table.  FWIW, the first examples I found don't use continue text, and our other Ped. line doesn't have it either, but I'll add to both.

**Comment by MarcSabatella** [comment]
Updated to add continue text (Ped.) to both Ped. lines.

File: mscore/menus.cpp [hunk 2]
```diff
@@ -870,8 +870,20 @@ Palette* MuseScore::newLinesPalette(bool basic)
...
+            pedal = new Pedal(gscore);
+            pedal->setLen(w);
+            pedal->setBeginText("<sym>keyboardPedalPed</sym>");
+            pedal->setContinueText("(<sym>keyboardPedalPed</sym>)");
+            pedal->setEndText("<sym>keyboardPedalUp</sym>");
+            Align align = pedal->endTextElement()->textStyle().align();
+            align = (align & AlignmentFlags::VMASK) | AlignmentFlags::HCENTER;
+            pedal->endTextElement()->textStyle().setAlign(align);
+            pedal->setLineVisible(false);
+            sp->append(pedal, QT_TRANSLATE_NOOP("Palette", "Pedal"));
```
Comment (lasconic) [inline mscore/menus.cpp]: If we move forward with the idea of a searchable palette, we will need more descriptive description for palette elements.

**Pull request merged**